src/components/PersonDetails/PersonDetails.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import cn from 'classnames'
+import { Person } from "../../types";
+import { PersonLink } from "../PersonLink/PersonLink";
+
+type Props = {
+  person: Person;
+  isSelected: boolean;
+  findPersonByName: (name: string | undefined) => Person | undefined;
+};
+
+export const PersonDetails: React.FC<Props> = ({person, isSelected, findPersonByName}) => {
+  const { slug, sex, born, died, motherName, fatherName } = person;
+
+  return (
+    <tr
+    key={slug}
+    data-cy="person"
+    className={cn({
+      'has-background-warning': isSelected,
+    })}
+  >
```

Comment:
Don't forget about code style
```suggestion
    <tr
      key={slug}
      data-cy="person"
      className={cn({
        'has-background-warning': isSelected,
      })}
    >
```